troposphere/settings/local.py.j2

```diff
@@ -64,20 +64,44 @@ OAUTH_CLIENT_CALLBACK = "{{ OAUTH_CLIENT_CALLBACK }}"
 SHOW_INSTANCE_METRICS = {{ SHOW_INSTANCE_METRICS }}
 {% if BADGES_ENABLED %}
 
+BADGES_ENABLED = True
+
 # Host for authenticating a badge request
 BADGE_HOST = "{{ BADGE_HOST }}"
 
 # Host for Badgekit API
 BADGE_API_HOST = "{{ BADGE_API_HOST }}"
 
+# The host that troposphere will fetch badge images from. Ex. {BADGE_HOST}/image_id
+BADGE_IMAGE_HOST = "{{ BADGE_IMAGE_HOST }}"
+
+# Host to give to OpenBadges API when adding to backpack
+BADGE_ASSERTION_HOST = "{{ BADGE_ASSERTION_HOST }}"
+
```

Comment:
It would be good to work with Steve on getting this configuration settings into the Secrets Repo for atmo-dev & atmobeta for verification with QA. Without that, this won't be visible to them.